### PR TITLE
Fix sorting issues

### PR DESCRIPTION
**Pull request by jsegaert**
Make sure to follow and check these boxes before submitting a PR! Thank you.

- [x] Explain the feature

This PR addresses various "weird sorting issues" that occur when using the "-s column" option with iocage list, iocage df and/or iocage snaplist.  Refer to issue #482 

- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: iocage/lib/ioc_common.py [hunk 1]
```diff
@@ -183,91 +232,106 @@ def sort_ip(ip, version="4"):
...
-    elif list_length == 6:
+    elif list_length == 5:
         _state = 0 if state[2] != "down" else 1
     else:
         _state = state
 
     # 0 is up, 1 is down, lame hack to get running jails on top.
-
-    return _state
+    # jails will be sorted by name within within state
```
Comment (skarekrow) [inline iocage/lib/ioc_common.py]: Too many within :P

**Review by skarekrow — CHANGES_REQUESTED** [review]
Just a very quick reveiw, this is a nice diff, it will require more testing from me before I merge

As always, thanks :)

**Comment by jsegaert** [comment]
Thank you for the heads-up.  Yes, there are quite a few changes.  Happy testing. :-)

**Review by skarekrow — APPROVED** [review]

**Comment by skarekrow** [comment]
Thanks <3

**Pull request merged**